docs/source/user_guide/troubleshooting.rst

```diff
@@ -0,0 +1,39 @@
+.. _user_guide_troubleshooting:
+
+===============
+Troubleshooting
+===============
+This section explains how to resolve the most common issues encountered with ``pydpf-post``.
+It also includes suggestions for improving scripts.
+
+Using the solution
+------------------
+
+Invalid UTF-8 warning/issue 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Trying to load a solution with: 
```

Comment:
Assume that you are using the following code to load a solution:

Reply:
It's harder to comprehend when they sentence is split by a code example. It's much better to split into two complete sentences.